lib/liquid/drop.rb

```diff
@@ -52,6 +52,10 @@ def to_liquid
       self
     end
 
+    def to_raw_value
```

Comment:
`to_liquid_value` vs `to_raw_value`?

Reply:
`to_liquid_value` is the value that will be stored in the registry.
`to_raw_value` is the value that will be used in a variable evaluation

Reply:
oops... `to_raw_value` has been renamed to `to_liquid_value`...
so it is now:

`to_liquid` is the value that will be stored in the registry.
`to_liquid_value` is the value that will be used in a variable evaluation